src/main/java/bio/terra/cli/app/utils/LocalProcessLauncher.java

```diff
@@ -76,6 +77,12 @@ public void launchProcess(
     }
   }
 
+  public Process launchSilentProcess(List<String> command, Map<String, String> envVars, Path workingDirectory) {
```

Comment:
add java doc for public method

unclear to me why this is called silent process.